tutorials/advanced/transforms_tutorial.py

```diff
@@ -303,6 +316,58 @@
 #
 #     trans = ax.get_xaxis_transform()
 #
+# .. _transforms-fig-scale-dpi:
+#
+# Plotting in physical units
+# ==========================
+#
+# Sometimes we want an object to be a certain physical size on the plot.
+# Here we draw the same circle as above, but in physical units.  If done
+# interactively, you can see that changing the size of the figure does
+# not change the offset of the circle from the lower-left corner,
+# does not change its size, and the circle remains a circle regardless of
+# the aspect ratio of the axes.
+
+fig, ax = plt.subplots(figsize=(5, 4))
+x, y = 10*np.random.rand(2, 1000)
+ax.plot(x, y*10., 'go')  # plot some data in data coordinates
+# add a circle in fixed-units
+circ = mpatches.Circle((2.5, 2), 1.0, transform=fig.dpi_scale_trans,
+                      facecolor='yellow', alpha=0.5)
+ax.add_patch(circ)
+plt.show()
+
+fig, ax = plt.subplots(figsize=(7, 2))
+x, y = 10*np.random.rand(2, 1000)
+ax.plot(x, y*10., 'go')  # plot some data in data coordinates
+# add a circle in fixed-units
+circ = mpatches.Circle((2.5, 2), 1.0, transform=fig.dpi_scale_trans,
+                      facecolor='yellow', alpha=0.5)
+ax.add_patch(circ)
+plt.show()
```

Comment:
Could it make sense to get rid of that strong green and pale yellow? After all it's the circle that is important here, not the  1000 scatter points.